packages/extensions/core/src/lib/configuration/configuration-loader.ts

```diff
@@ -246,7 +245,7 @@ export class ConfigurationLoader {
       configurationFiles[configFileUri] = await (await fsInputView.ReadStrict(configFileUri)).ReadData();
 
       const blocks = await this.parseCodeBlocks(await fsInputView.ReadStrict(configFileUri), await createView());
-      await addSegments(blocks, false);
+      await addSegments(blocks.reverse(), false);
```

Comment:
Seems like we shouldn't have to remember to reverse the list of blocks every time.  Is it possible that we could update `addSegments` to sort the segments based on whether they have logic fences?  How does reversing the lost of blocks help in this case?

Reply:
I think the issue is that right now it counts on the list of segments to be in order of high pri -> low pri. Now if when you do addSegemnts it reverse I feel it does something you didn't asked for. I am working on reorganizing all the config loading. See the other PR. Its quite a mess in general but as the rewrite I should have a better way to load a readme file and its code blocks